armi/reactor/components/__init__.py

```diff
@@ -186,6 +186,23 @@ def getBoundingCircleOuterDiameter(self, Tc=None, cold=False):
         """
         return 2 * math.sqrt(self.getComponentArea(cold=cold) / math.pi)
 
+    def getCircleInnerDiameter(self, Tc=None, cold=False):
+        """
+        Component is unshaped; assume it is circular and there is no ID (return 0.0).
+
+        Parameters
+        ----------
+        Tc : float, optional
+            Ignored for this component
+        cold : bool, optional
+            Ignored for this component
+
+        Notes
+        -----
+        Tc is not used in this method for this particular component.
```

Comment:
```suggestion
```

Reply:
That's an optional argument in this method, we don't need to call it out a second time.